File: openapi/code_samples/python/api_v1_organizations_{org_id}_roles_{role_name}/put.py
from scalekit.v1.roles.roles_pb2 import UpdateRole

scalekit_client.roles.update_organization_role(
    org_id="org_123",
    role_name="org_admin",
    role=UpdateRole(
        display_name="Org Admin (Updated)",
        description="Updated org role description"
    )
)

File: openapi/code_samples/java/api_v1_organizations_{org_id}_roles_{role_name}/put.java
UpdateOrganizationRoleResponse res = scalekitClient.roles().updateOrganizationRole(
    "org_123",
    "org_admin",
    UpdateOrganizationRoleRequest.newBuilder()
        .setRole(
            UpdateRole.newBuilder()
                .setDisplayName("Org Admin (Updated)")
                .setDescription("Updated org role description")
                .build()
        )
        .build()
);